### ISSUE TITLE
Getting ONLY_PREVIEW_ALLOWED error on all stories but Apple News has confirmed account is approved

### ISSUE THREAD

**Issue by hechinger** [alleyinteractive/apple-news#616]
Getting ONLY_PREVIEW_ALLOWED error on all stories but Apple News has confirmed account is approved

This happens on any stories we attempt to publish to Apple News but I pulled all the debugging info for this one.

https://hechingerreport.org/americans-think-state-funding-for-higher-ed-has-held-steady-or-risen-survey-finds/

Getting this error everytime I try to publish any story to Apple News. Have tried changing various settings but nothing seems to work.

There has been an error with the API: ONLY_PREVIEW_ALLOWED

**I emailed Apple News and they confirmed that the account is Approved.** 

Here is the full raw text of the article.

https://pastebin.com/k5KMQqbu

Here is the json of the article as exported from the plugin. (Note, when I try to upload any json from the plugin into Apple News I get an error that says the file is not a json file.)

https://pastebin.com/H3raj14Y

This is the error message I got from the debugging tool.

https://pastebin.com/YRSS5VNg

**Comment by Matt633** [comment]
@hechinger  I'm having the same problem.

@mslinnea I'm going to paste the API Notification error I get.  Any help would be greatly appreciated -- I haven't seen any dev responses from Alley in here. Are you guys actively supporting this? Thanks! 


API Response:
Array
(
   [headers] => Requests_Utility_CaseInsensitiveDictionary Object
       (
           [data:protected] => Array
               (
                   [server] => AppleHttpServer/c071afcf3886
                   [date] => Thu, 11 Jul 2019 18:59:20 GMT
                   [content-type] => application/json;charset=utf-8
                   [content-length] => 44
                   [request-id] => fce384d0-a40d-11e9-8351-4f79dff44095
                   [x-request-id] => fce384d0-a40d-11e9-8351-4f79dff44095
                   [x-test-source-environment] => prod
                   [x-b3-traceid] => 32999804efd63a4d
                   [strict-transport-security] => max-age=31536000
               )

       )

   [body] => {"errors":[{"code":"ONLY_PREVIEW_ALLOWED"}]}
   [response] => Array
       (
           [code] => 400
           [message] => Bad Request
       )

   [cookies] => Array
       (
       )

   [filename] => 
   [http_response] => WP_HTTP_Requests_Response Object
       (
           [response:protected] => Requests_Response Object
               (
                   [body] => {"errors":[{"code":"ONLY_PREVIEW_ALLOWED"}]}
                   [raw] => HTTP/1.1 400 Bad Request
Server: AppleHttpServer/c071afcf3886
Date: Thu, 11 Jul 2019 18:59:20 GMT
Content-Type: application/json;charset=utf-8
Content-Length: 44
Connection: close
Request-Id: fce384d0-a40d-11e9-8351-4f79dff44095
X-REQUEST-ID: fce384d0-a40d-11e9-8351-4f79dff44095
X-TEST-SOURCE-ENVIRONMENT: prod
X-B3-TraceId: 32999804efd63a4d
Strict-Transport-Security: max-age=31536000

{"errors":[{"code":"ONLY_PREVIEW_ALLOWED"}]}
                   [headers] => Requests_Response_Headers Object
                       (
                           [data:protected] => Array
                               (
                                   [server] => Array
                                       (
                                           [0] => AppleHttpServer/c071afcf3886
                                       )

                                   [date] => Array
                                       (
                                           [0] => Thu, 11 Jul 2019 18:59:20 GMT
                                       )

                                   [content-type] => Array
                                       (
                                           [0] => application/json;charset=utf-8
                                       )

                                   [content-length] => Array
                                       (
                                           [0] => 44
                                       )

                                   [request-id] => Array
                                       (
                                           [0] => fce384d0-a40d-11e9-8351-4f79dff44095
                                       )

                                   [x-request-id] => Array
                                       (
                                           [0] => fce384d0-a40d-11e9-8351-4f79dff44095
                                       )

                                   [x-test-source-environment] => Array
                                       (
                                           [0] => prod
                                       )

                                   [x-b3-traceid] => Array
                                       (
                                           [0] => 32999804efd63a4d
                                       )

                                   [strict-transport-security] => Array
                                       (
                                           [0] => max-age=31536000
                                       )

                               )

                       )

                   [status_code] => 400
                   [protocol_version] => 1.1
                   [success] => 
                   [redirects] => 0
                   [url] => https://news-api.apple.com/channels/f54af018-cc9c-4447-a32e-f74039504709/articles
                   [history] => Array
                       (
                       )

                   [cookies] => Requests_Cookie_Jar Object
                       (
                           [cookies:protected] => Array
                               (
                               )

                       )

               )

           [filename:protected] => 
           [data] => 
           [headers] => 
           [status] => 
       )

)

**Comment by kevinfodness** [comment]
The `ONLY_PREVIEW_ALLOWED` response you're getting is coming directly from Apple's API. Can you confirm that you're using the correct channel details? The only reason why this error should show is if the channel the plugin is pointing at isn't approved by Apple.

**Comment by brookenf** [comment]
@jomurgel Thank you for referring me to this issue. As others have stated above, I've confirmed with Apple that the account is valid. I also changed the Apple News keys and secret but I still receive the same error every time we publish an article. We're using the Auto-publish feature and we're still getting this issue. Currently running `1.4.3` to try and fix the issue from #669 since we're using the Classic Editor. That didn't seem to fix the issue, either.
